code/modules/admin/verbs/adminhelp.dm

```diff
@@ -141,3 +141,234 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 	feedback_add_details("admin_verb","AH") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 	return
 
+*/
+
+
+//This is a list of words which are ignored by the parser when comparing message contents for names. MUST BE IN LOWER CASE!
+var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","alien","as")
```

Comment:
use managed globals, check out global_vars/lists for examples, and put it in that folder somewhere

Reply:
Fixed.